### ISSUE TITLE
disk full when saving to storage - render

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#1201]
```
QuotaExceededError: Encountered disk full while committing transaction.
at Promise (chrome-extension://bnjglocicdkmhmoohhfkfkbbkejdhdgc/js/common/storage.js:369:31)
```

**Comment by tomholub** [comment]
Please try to see if there's a way to reproduce this (there is probably a chrome dev flag somewhere to simulate this).

View should already automatically render such errors to user - please verify that this is the case, and update `isSignificant` so that such errors would not get reported to us.

**Comment by michael-volynets** [comment]
Unfourtenetelly, there aren't any dev tools yet, Chromium team is working on it now (https://bugs.chromium.org/p/chromium/issues/detail?id=945786).
I also tried to do it another way. I created a folder and mounted it with size of 10Mb (it can contain 10mb maximum), after that, I ran google chrome there using this command: 'google-chrome --user-data-dir=<pathToThatFolder>' but what happened next, when I installed FlowCrypt and I was using it for about 3-4min browser is totally crashed.

I think it will be better to wait until they finish working on that dev. feature and then we will be able to test it properly.
